pages/products/success-tokens.tsx

```diff
@@ -0,0 +1,159 @@
+        The <b>call option</b> may pay investors an additional amount of your
+        DAO token, but only if the price of your token has gone up in the
+        meantime. This means these strategic investors are only given any bonus
+        if the DAO is doing well.
+      </>,
+      "It is easy to see why tokenholders like this model: in this event, everyone voting on this proposal today will have also seen the value increase. In either event, the DAO has increased its treasury and gained a strategic partner.",
+      "Finally, investors gain the kind of risk profile they are looking for. Namely, exposure to upside. If the token value rallies 2x or 3x over the strike price, they will benefit more than if they had purchased at a discount. It's a win-win-win.",
+    ],
+    link: {
+      text: "Learn More",
+      href: "https://docs.umaproject.org/success-tokens/summary",
+    },
+  },
+  {
+    type: Section.Tldr as Section.Tldr,
+    image: {
+      src: "/images/products/UMA-TLDR.svg",
+      alt: "TLDR",
+    },
+    details:
+      "Instead of paying the venture capitalist investor upfront via a discount, the Success Token pays the VC a bonus only if the protocol is successful in achieving its goals.",
+  },
+  {
+    type: Section.Starting as Section.Starting,
+    title: "How does it work?",
+    sections: [
+      {
+        title: "Sale Design",
+        details:
+          "Your DAO decides on the parameters. Our team of financial engineers can offer guidance as you choose your strike price, expiration date, and total sale size.",
+      },
+      {
+        title: "Mint Tokens",
+        details:
+          "Your DAO deposits tokens into the contract and mints the success tokens. UMA's contracts have been audited, held over $200m in collateral, and have been exploit-free since inception over a year ago.",
+      },
+      {
+        title: "Raise funds for Treasury",
+        details:
+          "Your DAO sells these tokens to strategic investors. Investors are looking for upside in DeFi projects and this sale type has been tested and investor approved.",
+      },
+    ],
+    link: {
+      text: "Contact Us",
+      href: "https://discord.umaproject.org/",
+    },
+  },
+  {
+    type: Section.Partners as Section.Partners,
+    title: "Industry Examples",
+    partners: [
+      {
+        image: {
+          src: "/images/products/UMA-Sushi.svg",
+          alt: "SushiSwap",
+        },
+        title: "SushiSwap Proposal",
+        link: {
+          text: "Read it here",
+          href: "https://forum.sushi.com/t/withdrawn-sushi-phantom-troupe-strategic-raise/4554/256",
+        },
+      },
+      {
+        image: {
+          src: "/images/products/UMA-Shapeshift.svg",
+          alt: "Shapeshift",
+        },
+        title: "Shapeshift Proposal",
+        link: {
+          text: "Read it here",
+          href: "https://docs.google.com/spreadsheets/d/1wRYgPVIjEyk6ADi1f0IKhKXoN3iaZRG7v3Iza-HhNIw/edit#gid=53260911",
```

Comment:
Update this link to: https://forum.shapeshift.com/t/success-token-proposal-discussion/599